pyais/messages.py

```diff
@@ -784,8 +787,8 @@ class MessageType17(Payload):
     repeat = bit_field(2, int, default=0)
     mmsi = bit_field(30, int, from_converter=from_mmsi, to_converter=to_mmsi)
     spare_1 = bit_field(2, int, default=0)
-    lon = bit_field(18, int, from_converter=from_course, to_converter=to_course, default=0)
-    lat = bit_field(17, int, from_converter=from_course, to_converter=to_course, default=0)
+    lon = bit_field(18, float, from_converter=from_course, to_converter=to_course, default=0)
```

Comment:
should be from_lat_lon, to_lat_lon
Are there any test cases for these messages where data could be checked?

Reply:
No, latitude and longitude are in units of a tenth of a minute. [Source](https://gpsd.gitlab.io/gpsd/AIVDM.html#_type_17_dgnss_broadcast_binary_message).

There is a testcase for it: https://github.com/M0r13n/pyais/blob/e833f46cbaa157aa022113c10a5630c002104459/tests/test_decode.py#L323